src/main/java/emu/grasscutter/game/managers/chat/ChatManager.java

```diff
@@ -156,6 +151,8 @@ public void sendPrivateMessage(Player player, int targetUid, String message) {
             target.sendPacket(packet);
             putInHistory(targetUid, player.getUid(), packet.getChatInfo());
         }
+        // Check if command
+        tryInvokeCommand(player, target, message);
```

Comment:
Can't do a direct suggestion because L150 is out of scope
```java
        // Check if command
        boolean isCommand = tryInvokeCommand(player, target, message);

        // Send to target player if message is not a command
        if ((target != null) && (!isCommand)) {
            target.sendPacket(packet);
            putInHistory(targetUid, player.getUid(), packet.getChatInfo());
        }
```